src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1156,4 +1157,17 @@ function ($a) {
             ->end()
         ;
     }
+
+    private function addRobotsIndexSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->booleanNode('disallow_search_engine_index')
+                    ->info('Ask search engines to avoid indexing your application. Enabled by default when debug is enabled.')
```

Comment:
I would remove the "Ask search engines to avoid indexing your application. " part as the option name is quite explicit.

Reply:
Done, as well as the changelog